cogs/wordfilter/wordfilter.py

```diff
@@ -550,20 +570,100 @@ async def on_message(self, msg):
     async def on_message_edit(self, msg, newMsg):
         await self.checkWords(msg, newMsg)
 
+    ############################################
+    # COMMANDS - Usage Statistics #
+    ############################################
+    @stat.command(name="rawusage")
+    @commands.guild_only()
+    @checks.mod_or_permissions(manage_messages=True)
```

Comment:
Can you double check and see if we can actually remove this? IIRC since the root command `wordfilter` has those two decorators already, these commands will automatically do the check.

Reply:
Yeah, not needed